src/index.js

```diff
@@ -2,9 +2,23 @@
  * Id is an object representation of a peer Id. a peer Id is a multihash
  */
 
+var fs = require('fs')
 var multihashing = require('multihashing')
 var base58 = require('bs58')
-var keypair = require('keypair')
+var forge = require('node-forge')
+var protobuf = require('protocol-buffers')
+
+var isNode = !global.window
+
+// protobuf read from file
+var messages = isNode ? protobuf(fs.readFileSync(__dirname + '/../pb/crypto.proto')) : protobuf(require('buffer!./../pb/crypto.proto'))
+
+// for some reason webpack can only find forge at forge.forge().someFunction()
+// browser should be able to just use forge.someFunction()
+// this is only happening when js-ipfs bundles peer-id module
+/* if(!isNode){
+  forge = forge.forge()
+} */
```

Comment:
it could be unfortunately haha. This comment is part of a weird bug that I was talking to @dignifiedquire about. When noParsing the forge build file in the js-ipfs web tests, the forge functions were exposed at forge.forge().function instead of forge.function. This is not true for the peer-id tests where forge is at the normal forge.function. Not sure if this will continue in js-ipfs or why it is happening. Using that commented out code is a work around but it needs to be consistent with the js-peer-id or it won't work for both versions. I'll test more with js-ipfs to see if it's still happening